databroker/experimental/example_dataframe.py

```diff
@@ -0,0 +1,34 @@
+import numpy
```

Comment:
this should probably be either deleted or moved into tests

Reply:
Deleted it